04-materials/05-working-on-your-own.md

```diff
@@ -24,14 +16,93 @@ TBD
 We will be using the [official template](https://github.com/jupyterlab/extension-template) as the starting point to creating your own extension.
 :::
 
-## 📋 Extension Specs
-TBD
+## 🎯 Session Format
+This is an **open exploration session**. You'll choose a project that interests you and work on it with instructor support. Don't worry about finishing - the goal is to practice the development workflow and problem-solving.
+
 
-## 🐛 Common issues and debugging tips
+## 💡 Extension Ideas
+[JupyterLab Extension Ideas](https://github.com/jupyterlab/jupyterlab/issues?q=sort%3Aupdated-desc+state%3Aopen+label%3A%22tag%3AExtension+Idea%22)
+
+## Community Extensions
+[Jupyterlab-contrib](https://jupyterlab-contrib.github.io/extensions.html)
+
```

Comment:
Would you feel comfortable including a link to JupyterLab Community Marketplace? https://labextensions.dev
This can help them get an idea of what extensions are out there